src/core/mempool.rs

```diff
@@ -299,13 +361,14 @@ const MEMPOOL_INITIAL_SCHEMA: &'static [&'static str] = &[
     );
     "#,
     "CREATE INDEX by_txid ON mempool(txid);",
+    "CREATE INDEX by_txid_and_height ON mempool(txid,height);",
     "CREATE INDEX by_sponsor ON mempool(sponsor_address, sponsor_nonce);",
     "CREATE INDEX by_origin ON mempool(origin_address, origin_nonce);",
     "CREATE INDEX by_timestamp ON mempool(accept_time);",
     "CREATE INDEX by_chaintip ON mempool(consensus_hash,block_header_hash);",
 ];
 
-const MEMPOOL_SCHEMA_2: &'static [&'static str] = &[
+const MEMPOOL_SCHEMA_COST_ESTIMATOR: &'static [&'static str] = &[
```

Comment:
While `MEMPOOL_SCHEMA_2` isn't particularly descriptive, the name _does_ correspond to the DB version, which is helpful. Maybe these schema arrays should be named like `MEMPOOL_SCHEMA_2_COST_ESTIMATOR`?

Reply:
Sure; done.